gpiozero/pins/mock.py

```diff
@@ -413,3 +385,34 @@ def tx_word(self, value, bits_per_word=None):
             bits = reversed(bits)
         self.tx_buf.extend(bits)
 
+
+class MockFactory(LocalPiFactory):
+    def __init__(self, revision='a21041', pin_class=MockPin):
```

Comment:
`revision` probably should be settable by env-vars (and it'd be trivial to add; though I'm not sure if any tests currently depend on the default?).

Whether `pin_class` should be ... erm ... not sure. It's a bit of a hodge-podge at the moment anyway given the pin class can be specified in numerous places (the factory constructor, after construction by simply re-writing the `pin_class` field, or even when calling the `pin()` method where the class is an optional parameter). Is there much difference between writing, say, `$ MOCK_PIN_CLASS=MockPWMPin python` and `>>> factory = MockFactory(pin_class=MockPWMPin)` or am I missing a use-case here?